web/client/translations/data.it-IT.json

```diff
@@ -1125,12 +1125,13 @@
             "changePwd": "Cambia Password",
             "newPwd": "Nuova Password",
             "retypePwd": "Conferma Password",
-            "passwordMinlenght": "La password deve essere lunga almeno {minSize} caratteri",
+            "passwordMinlenght": "La password deve essere lunga almeno {data} caratteri",
             "passwordCheckFail": "Le due password non corrispondono",
             "passwordInvalid": "Password non valida",
             "username": "Username",
             "password": "Password",
-            "passwordMessage": "La password deve contenere almeno 6 caratteri",
+            "passwordMessage": "La password deve contenere almeno 6 caratteri. Personaggi ammessi !,@,#,$,%,&,*,_",
```

Comment:
Personaggi is the italian for character, in the sens of "personality". Please check with reverse translation that this error is not present in the other translations.

Reply:
@offtherailz made updates according to your review.